packages/wrangler/src/utils/shell-quote.ts

```diff
@@ -1,6 +1,10 @@
 import shellquote from "shell-quote";
 
-export const quote = shellquote.quote;
+export const quote = function (args: (string | number | boolean)[]) {
+	const stringArgs = args.map((arg) => String(arg));
```

Comment:
As I mentioned above, is this actually needed? 😮 

PS: the code duplication here isn't the best... I guess it'd be too much effort (with maybe currently too little benefit?) to create a new internal package with utilities that can be used across all packages in the monorepo?

Reply:
I want to pull this out into a package but I was waiting on https://github.com/cloudflare/workers-sdk/pull/4038 to be merged

I will raise with the team and try to get it merged soon